packages/sources/vue/src/components/structure/VtmnListItem/VtmnListItem.vue

```diff
@@ -40,16 +51,38 @@ export default /*#__PURE__*/ defineComponent({
     :aria-disabled="disabled"
     v-bind="$attrs"
   >
-    <div v-if="$slots['start-visual']" class="vtmn-list_start-visual">
-      <slot name="start-visual" />
-    </div>
+    <template v-if="href">
+      <a
+        class="vtmn-list__link"
+        :href="href"
+        :target="target"
+        :rel="computeRel(target, rel)"
+        :aria-disabled="disabled"
```

Comment:
Same here should have a way to set an `aria-label`

Reply:
Same as above